tooling/scripts/dl-monaco.ts

```diff
@@ -4,10 +4,10 @@ import path from 'node:path';
 import fetch from 'node-fetch';
 
 /** @link https://typescript.azureedge.net/indexes/releases.json */
-const TYPESCRIPT_VERSION = '5.3.3';
+const TYPESCRIPT_VERSION = '5.4.5';
```

Comment:
i see the vercel build failed, could we only do one change at a time?

```suggestion
const TYPESCRIPT_VERSION = '5.3.3';
```

would require you to rerun the script

Reply:
ugh fine ill just make this a url change